### ISSUE TITLE
Latest release on 6 Dec 2016, is this project still alive?

### ISSUE THREAD

**Issue by meglio** [kakserpom/phpdaemon#292]
Please advise if it's still maintained and whether it's safe to use it? If no, which alternatives would you recommend?

**Comment by kakserpom** [comment]
Yes, it's still alive and maintained. We're going to make a lot of updates soon.

**Comment by sashok1337** [comment]
Just wondering is there any updates on project status?

**Comment by kakserpom** [comment]
Well, I've switched away from PHP to other (better) technologies.
The project is in a maintenance mode. It means that I will fix critical issues, but I am unlikely to develop new features.
